cmd/edgectl/aes_install.go

```diff
@@ -620,6 +624,25 @@ func (i *Installer) Perform(kcontext string) Result {
 
 		i.clusterinfo.CopyChartValuesTo(chartValues)
 
+		reportedVersion := "unknown"
+		if Version != UnknownVersion {
+			reportedVersion = Version
+			// replace all the forbidden chars in the value by a `-`
+			for _, forbidden := range []string{",", " ", "\\", "/", "(", ")", "\"", "*", "^", "$", "#", "@", "\n", "\t"} {
+				reportedVersion = strings.ReplaceAll(reportedVersion, forbidden, "-")
+			}
+		}
+
+		// Save some environment info as Helm values
+		currentTime := time.Now()
+		chartValues[defHelmValueRootEdgectlInfo] = map[string]interface{}{
+			"edgectl/install-cluster-type": i.clusterinfo.name,
```

Comment:
I'd leave `install-cluster-type` aside... I don't think it's relevant to either Ambassador or the ops people behind the installation. Plus, what if we get it wrong? It's one thing to show it in egectl logs but not a definitive answer.

Reply:
I think that reporting the environment where Ambassador has been installed could be valuable information from a statistical point of view. Knowing that users are using GKE or Minikube for trying Ambassador can give us an interesting insight, even if this information is not 100% accurate. Maybe @bnfb can comment on this.

Reply:
We can already extract this information in metriton using a simple join on 2 fields if we want to build such a view. I think we should limit the number of labels to prevent bloating the pod template with things we can infer otherwise and make sure every instrumented label serves a purpose. Their usage and intent should also be documented.

Instrumenting the cluster-type could be dangerous as Ambassador needs to remain agnostic of the platform (and support them all without if==gke, if==aws)

Reply:
If we can report that information in metriton then great: let's have a metq report that does that and call it good. But the goal is to get the report.